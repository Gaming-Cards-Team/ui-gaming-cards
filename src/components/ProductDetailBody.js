import React from "react";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";
import ProductText from "./ProductText";


const imageStyle = {
  textAlign: "right",
};
const productTextStyle = {
  color: "#2d963f",
  fontWeight: "bold",
};

const ProductDetailBody = () => {
  return (
    <div>
    <Row>
      <Col xs={5} style={imageStyle}>
        Imagen
      </Col>
      <Col xs={7} style={productTextStyle}>
            <ProductText />
      </Col>
    </Row>
  </div>

  );
};

export default ProductDetailBody;
