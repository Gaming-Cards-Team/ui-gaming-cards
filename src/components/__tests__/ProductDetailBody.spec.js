import React from 'react';
import ProductDetailBody from '../ProductDetailBody';

import renderer from 'react-test-renderer';

describe('Test Product Detail Body component', () => {
    it('should validate the component', () => {
        const productDetailBody = renderer.create(<ProductDetailBody />).toJSON();

        expect(productDetailBody).toMatchSnapshot();
    });
});