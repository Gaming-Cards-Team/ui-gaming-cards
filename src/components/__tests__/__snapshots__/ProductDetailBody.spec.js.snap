// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Product Detail Body component should validate the component 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-5"
      style={
        Object {
          "textAlign": "right",
        }
      }
    >
      Imagen
    </div>
    <div
      className="col-7"
      style={
        Object {
          "color": "#2d963f",
          "fontWeight": "bold",
        }
      }
    >
      <div>
        Nombre de la tarjeta Mini detalle
        <div
          className="row"
        >
          <div
            className="col-6"
            style={
              Object {
                "textAlign": "right",
              }
            }
          >
            Atributos
          </div>
          <div
            className="col-6"
            style={
              Object {
                "color": "#2d963f",
                "fontWeight": "bold",
              }
            }
          >
            Desripcion
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
